### DISCUSSION TITLE
DEV SESSION [TUESDAY] 14 May -- 2030 UTC+8/1430 CEST/1330 BST/ 0830 ET

### DISCUSSION THREAD

**Discussion by tpike3** [mesa/mesa#2127] [General]
Where: [via Matrix](https://matrix.to/#/#project-mesa:matrix.org)

**Proposed Agenda**

- GSoC discussion
  - GSoC Mentor Summit: https://sites.google.com/view/2024mentorsummit/home
- PRs:
    - #2092
    - #2096
    -  #2100 
    - #2118
- Mesa 3.0
- Mesa-Geo
- Paper?
- Epistemix - https://www.epistemix.com/ 

cc @tpike3, @EwoutH, @Corvince , @wang-boyu , @rht, @quaquel

**Comment by jackiekazil** [comment]
High level notes from this meeting: 

Discussion: Mesa and Mesa-Geo tutorial 

**GSoC Discussion:** 
 * Dong: Can I extend for two weeks - and we said would 
 * Harsh: need time, not compute
 * GSoC Summit: Jackie will commit to going. 

**Paper**
* Andrew - thinks it is a good time for another Mesa paper -- Since the last one was 2020. 
* Papers
  * Mesa 3.0 paper -- Summarize updates. (Boyu)
  * Main Mesa citation -- JOSS paper (Jackie) 
* GSOC Papers? 

  
_More discussion to happen around papers._

No tickets were discussed because of time constraints.

**Reply by EwoutH** [reply]
Thanks for the notes. I agree on the papers. I was also discussing this with some other maintainers a while ago. I would be interested in contributing on the papers, especially on the PRs I worked on (AgentSet, PropertyLayers, scheduling).

I would frame it as a “Mesa 3.0” paper.

**Reply by wang-boyu** [reply]
Thanks @EwoutH. I was hoping to talk to you in this dev session about the paper. Is there a link to your past discussions about it? Will you be there in the next dev session?

**Reply by jackiekazil** [reply]
@EwoutH There are two papers here. I updated the notes to clarify. 
One is the Mesa 3.0 paper, which is a summary of updates.  The other one is Joss paper, which simply becomes the main citation for Mesa, so we have a consistent citation that people point, which holds the basic concepts of Mesa and that can been updated if needed.

**Reply by EwoutH** [reply]
It was somewhere on matrix, probably in private, don't remember where exactly. But the main idea aligns with the Mesa 3.0 paper Jackie describes.

**Comment by jackiekazil** [comment]
Closing this for clean up.
